### PR TITLE
fix(classic-eth-flow): wrap no longer clears input amounts

### LINKED ISSUE #3421: Classic ETH flow is broken when a token requires approval
**The issue is described for a situation when a traded amount < approved amount. 
But it appears that it is impossible to proceed with the classic ETH flow when traded WETH amount > approved amount (indeed requires approval).** 
image
---------------------------------

**How to Reproduce**
1. Approve WETH token partially (e.g. 1 WETH)
2. Make sure that WETH balance is less than 0.5 (otherwise please unwrap it)
3. Specify a token pair: 0.5 ETH per any other token
4. Switch to Classic WETH flow and Wrap ETH to WETH

**AR**: when a token is wrapped, 'Approve' token modal appears. However, there is no need in approving token as trading balance is less than an approved token. 
BE shows an error when press on Approve

**Expected behavior**
No need to approve a token if trading amount < approved amount. 
A Confirm Swap modal should appear after Wrap modal

**Screenshots**
no need in approval - enough
not working

### PR DESCRIPTION
**Pull request by alfetopito**
# Summary

Fix #3421

Classic eth flow works again!

  # To Test

1. Select native as sell and another token as buy
2. Input the sell amount less than what you have in wrapped native
3. Click on `wrap` button at the bottom of the SWAP form 
image
4. Click on `wrap and swap` button at the bottom 
image
5. Wrap
* Should wrap
6. After wrapped, approve (if needed)
* Should approve if needed
7. Swap
* Should swap with wrapped native

### CODE REVIEW

File: apps/cowswap-frontend/src/legacy/hooks/useWrapCallback.ts [hunk 1]
```diff
@@ -85,9 +71,6 @@ export async function wrapUnwrapCallback(
     })
     useModals && closeModals()
 
-    // Clean up form fields after successful wrap/unwrap
-    updateTradeState({ ...tradeState, inputCurrencyAmount: '', outputCurrencyAmount: '' })
-
```
Comment (alfetopito) [inline apps/cowswap-frontend/src/legacy/hooks/useWrapCallback.ts:90]: This is the fix.

After the wrap, the input amounts were reset, clearing the trade state, short-circuiting the flow from then on.

**Review by shoom3301 — APPROVED** [review]
https://goerli.etherscan.io/tx/0x582718325dd07aab7cd8a5d52ac3ad52ff9ae3105d848f9e9e734778b89700fa

https://dev.explorer.cow.fi/goerli/orders/0x9972f3fd07022dcb7c345b8dceafc5aec8e145abb886a165a2ffe1f17a6ed012fb3c7eb936caa12b5a884d612393969a557d4307658163ef?tab=overview

Tested, works great!

**Review by anxolin — APPROVED** [review]
@elena-zh could you test this one pls?

**Comment by elena-zh** [comment]
@alfetopito , great job!

**Review by elena-zh — APPROVED** [review]

**Pull request merged**